File: microservices/investment/src/main/java/com/illunex/invest/investment/persistence/entity/Judge.java
package com.illunex.invest.investment.persistence.entity;

import lombok.*;

import javax.persistence.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "judge")
public class Judge {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long idx;
    Long companyIdx;
    Long userIdx;
    String name;
    String rank;
    String company;
    String imgUrl;
    String comment;
    String status;
    Integer point;
}

File: microservices/invest/src/main/kotlin/com/illunex/invest/invest/persistence/entity/Judge.kt
package com.illunex.invest.invest.persistence.entity

import javax.persistence.*

@Entity
@Table(name = "judge")
data class Judge(
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    var idx: Long? = null,
    var userIdx: Long? = null,
    var name: String? = null,
    var rank: String? = null,
    var imgUrl: String? = null
) {
    constructor() : this(null, null, null, null, null )
}

